apps/design-system-docs/docs/components/packages/wp-components/icon-link/index.tsx

```diff
@@ -1,16 +1,19 @@
 import { Button } from '@wordpress/components';
-import { IconFigma } from './icon-figma';
 import { IconStorybook } from './icon-storybook';
 
-export const IconLink = ( { href, type }: { href: string; type: 'figma' | 'storybook' } ) => {
+const ICONS = {
+	storybook: <IconStorybook />,
+};
+
+export const IconLink = ( { href, type }: { href: string; type: 'storybook' } ) => {
```

Comment:
Maybe we can type this to reference keys of the object, both to be consistent with how we're using it in implementation, and to limit the number of references we need to update if we add / remove?

```suggestion
export const IconLink = ( { href, type }: { href: string; type: keyof typeof ICONS } ) => {
```

Reply:
Right cebf295 ✅